src/app/public/src/modules/sidebar/sidebar-wrapper.component.ts

```diff
@@ -1,85 +1,72 @@
-import { Component, Input, OnInit, Renderer2, OnDestroy, ElementRef, AfterViewInit } from '@angular/core';
-import { Subject } from 'rxjs';
+import { Component, Input, Renderer2, OnDestroy, AfterViewInit } from '@angular/core';
+import { Subscription } from 'rxjs/Subscription';
+import { SkyMediaQueryService, SkyMediaBreakpoints } from '@blackbaud/skyux/dist/core';
+
 import { StacheNavLink } from '../nav';
-import { StacheOmnibarAdapterService, StacheWindowRef } from '../shared';
+import { StacheWindowRef } from '../shared';
 
-const WINDOW_SIZE_MID: number = 992;
 const CONTAINER_SIDEBAR_CLASSNAME: string  = 'stache-container-sidebar';
+let nextUniqueId = 0;
 
 @Component({
   selector: 'stache-sidebar-wrapper',
   templateUrl: './sidebar-wrapper.component.html',
   styleUrls: ['./sidebar-wrapper.component.scss']
 })
-export class StacheSidebarWrapperComponent implements OnInit, OnDestroy, AfterViewInit {
+export class StacheSidebarWrapperComponent implements  OnDestroy, AfterViewInit {
   @Input()
   public sidebarRoutes: StacheNavLink[];
 
-  public sidebarOpen: boolean = false;
+  public isOpen: boolean = false;
```

Comment:
This is probably my fault, but we'll need to keep this variable named `sidebarOpen` to avoid breaking changes.

Reply:
👍